### DISCUSSION TITLE
Hard drive gets full fast. How do i delete the cache or set it up to automatically delete itself?

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#17290] [Q&A]
After about a week my mastodon stops working and i see a screen that says error with an elephant hitting a computer. 

I found the problem is that the hard drive fills up fast. I am assuming it saves everything from the federated as i dont have any users other than myself.

How can i set it up to delete the cache after a certain amount of days or amount of GB

**Comment by YJSoft** [comment]
https://docs.joinmastodon.org/admin/tootctl/#media

There are `tootctl media remove` command which will remove remote cached media.

**Reply by ghost** [reply]
i tried that and i get the follow error message

Command 'tootctl' not found, did you mean: 
command 'bootctl' from deb systemd

**Reply by YJSoft** [reply]
> The command-line interface of Mastodon is an executable file called tootctl residing in the bin directory within the Mastodon root directory. You must specify which environment you intend to use whenever you execute it by specifying the RAILS_ENV environment variable. Unless you are a developer working on a local machine, you need to use RAILS_ENV=production. If you are sure that you will never need another environment (for development, testing, or staging), you can add it to your .bashrc file for convenience, e.g.:
> 
> echo "export RAILS_ENV=production" >> ~/.bashrc
> If so, you won’t need to specify it each time inline. Otherwise, calls to tootctl will usually go like this, assuming that the Mastodon code is checked out in /home/mastodon/live:
> 
> cd /home/mastodon/live
> RAILS_ENV=production bin/tootctl help

https://docs.joinmastodon.org/admin/tootctl/

tootctl command is located at bin folder. you should use like above.

**Reply by ghost** [reply]
I tried it and it would not work then i found out i also need to log on to the mastodon admin account. Like this:

sudo su - mastodon

thanks for the info! i cleared the cache like your instructions

**Reply by ghost** [reply]
My database size is already 500MB. It grows by 500MB every week. I only have 1 user and he doesn't post anything. 

Im guessing its the federated database. 

How do i disable it saving so much. At that rate i will need tons of space in a year
